coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -57,11 +58,21 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         filename = source_range.file
 
         ignore_diff = Diff(original_file_dict[filename])
-        ignore_diff.change_line(
-            source_range.start.line,
-            original_file_dict[filename][source_range.start.line-1],
-            original_file_dict[filename][source_range.start.line-1].rstrip() +
-            '  ' + ignore_comment)
+
+        original_line = str(original_file_dict[filename][
+                        source_range.start.line-1].rstrip())
+
+        # Add the new ignore using 'and' with the previous ignore.
+        if re.search(r'Ignore.+Bear', original_line):
+            changed_line = re.sub(r'(.+Ignore\s)(.+)', r'\1' +
+                                  result.origin + r' and \2', original_line)
```

Comment:
Alright.

Reply:
@Makman2 there is no need to use ``re.escape`` on ``result.origin`` because ``result.origin`` is a string, not a pattern, if I do 
```
In [2]: re.escape('Ignore PylintBear (W505)')
Out[2]: 'Ignore\\ PylintBear\\ \\(W505\\)'
```